llvm/test/CodeGen/NVPTX/param-vectorize-device.ll

```diff
@@ -150,8 +150,8 @@ define dso_local void @caller_St4x3(ptr nocapture noundef readonly byval(%struct
   ; CHECK:       )
```

Comment:
Perhaps it's good opportunity to convert it to automatic checks, too. We do want to see that the values are passed in correct order and not just in a right kind of register matched by a regex.